app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
It's not clear to me what criteria IE/Edge uses to determine hardware acceleration is suitable.  I want to have evidence that this is actually needed for IE/Edge.  Create a layer is not cheap and we should avoid doing it if it's not needed.